### PR TITLE
Fix/fix bg search input

### PR DESCRIPTION
**Pull request by dmitriy-uvin**

### CODE REVIEW

File: client/src/store/modules/i18n/en.js [hunk 1]
```diff
@@ -340,6 +340,7 @@ export default {
     YOU_CAN_CHANGE_LANGUAGE_TO: 'Your can change language to',
     DISPLAY_LANGUAGE: 'Display language',
     MY_EVENTS: 'My Events',
+    UNAUTHENTICATED_ERROR: 'Session time is expired! SignIn again, please!',
```
Comment (yevhen-turovets) [inline client/src/store/modules/i18n/en.js:343]: Change "SignIn" on "Sign In"

File: client/src/store/modules/i18n/en.js [hunk 2]
```diff
@@ -340,6 +340,7 @@ export default {
     YOU_CAN_CHANGE_LANGUAGE_TO: 'Your can change language to',
     DISPLAY_LANGUAGE: 'Display language',
     MY_EVENTS: 'My Events',
+    UNAUTHENTICATED_ERROR: 'Session time is expired! Sign In again, please!',
```
Comment (yevhen-turovets) [inline client/src/store/modules/i18n/en.js:343]: "SignIn" change on "Sign In"

**Review by aLLeXUs — CHANGES_REQUESTED** [review]
Seems the branch containing changes from another branch. Please merge `develop` branch to make them disappear.

**Review by yevhen-turovets — APPROVED** [review]

File: client/src/components/auth/SignUp.vue [hunk 3]
```diff
@@ -81,6 +81,14 @@
...
                 </VBtn>
             </VForm>
+            <VBtn
+                height="44"
+                depressed
+                class="signup-button  primary"
+                @click="onSignUp"
+            >
+                {{ lang.SIGN_UP }}
+            </VBtn>
```
Comment (aLLeXUs) [inline client/src/components/auth/SignUp.vue:91]: Do we really need this? I see a similar button a few lines above.

Comment (dmitriy-uvin) [reply client/src/components/auth/SignUp.vue]: No, we don't.

**Review by aLLeXUs — APPROVED** [review]

**Pull request merged**